lib/compiler-singlepass/src/codegen_x64.rs

```diff
@@ -1724,7 +1729,7 @@ impl<'a> FuncGen<'a> {
 
     pub fn get_state_diff(&mut self) -> usize {
         if !self.machine.track_state {
-            return std::usize::MAX;
+            return crate::lib::std::usize::MAX;
```

Comment:
I'm curious, can we just do `use crate::lib::std` at the top of the file and have this work? My understanding is that explicit imports take precedence over the prelude. That said, I don't actually know if `std` is considered part of the prelude or its own crate.

Reply:
That works and looks way better!